lib/v4/migration-helpers/update-folder-structure.js

```diff
@@ -0,0 +1,25 @@
+const { resolve, join, basename } = require('path');
+const { logger } = require('../../global/utils');
+
+const moveDir = (appPath, folder, srcDir) => {
+	try {
+    await fs.copy(join(appPath, folder), srcDir);
+  } catch (error) {
+    logger.info(`${basename(appPath)}/${folder} not found, skipping...`);
+  }
```

Comment:
I like this log but ideally it would log before all the logs at the end of `lib/v4/migration-helpers/update-api-folder-structure.js`